code/upaya/app/models/app_setting.rb

```diff
@@ -0,0 +1,21 @@
+require 'feature_management'
+
+class AppSetting < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true
+
+  validates :value,
+            inclusion: { in: %w(0 1), message: :invalid },
+            presence: true, if: :boolean?
+
+  def self.registrations_enabled?
+    find_by_name('RegistrationsEnabled').try(:value) == '1'
+  end
+
+  def self.registrations_disabled?
+    !self.registrations_enabled?
+  end
+
+  def boolean?
+    name =~ /RegistrationsEnabled/
+  end
+end
```

Comment:
Is this something we know for sure we will be using or should we wait before importing it?